### PR TITLE
SG friends

### PR DESCRIPTION
**Pull request by valentinludu**

### CODE REVIEW

File: packages/utils/src/cuid.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { init } from '@paralleldrive/cuid2';
+
+/**
+ * Create by default a cuid with length 10
+ */
+export const createCuid = init({
```
Comment (mattcasey) [inline packages/utils/src/cuid.ts:6]: this should probably just live in strings util

File: apps/scoutgametelegram/lib/session/getSession.ts [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
 import { cookies } from 'next/headers';
 
 import { getIronOptions } from './getIronOptions';
 
+export type SessionData = {
+  anonymousUserId?: string;
+  scoutId?: string;
+};
+
+const baseUrl = 'https://5afa-2a02-2f04-8212-300-693c-507e-16d1-6b98.ngrok-free.app';
```
Comment (mattcasey) [inline apps/scoutgametelegram/lib/session/getSession.ts:12]: I know it's a WIP but this probably should not be hard-coded. usually this is passed in as process.env.DOMAIN and u could just set that in your .env.local

Comment (valentinludu) [reply apps/scoutgametelegram/lib/session/getSession.ts:12]: I will clean this up. This was just for testing.

File: apps/scoutgametelegram/components/friends/components/InviteButtons.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import { useUser } from 'components/layout/UserProvider';
+
+export function InviteButtons() {
+  const { user } = useUser();
+  const [copied, setCopied] = useState('');
+  const referral = user?.referralCode;
+  const url = `https://t.me/ScoutGameXYZBot/start?startapp=${referral}`;
+  const text = encodeURIComponent('Play ScoutGame with me!');
+
+  const shareUrl = `https://t.me/share/url?url=${url}&text=${text}`;
```
[the remark this answers is not in the document]
Comment (valentinludu) [reply apps/scoutgametelegram/components/friends/components/InviteButtons.tsx:17]: Updated!

File: packages/utils/src/cuid.ts (hunk 1 above)
Comment (valentinludu) [reply packages/utils/src/cuid.ts:6]: Done!

The package is 35kb.

**Review by mattcasey — APPROVED** [review]

**Pull request merged**